h2o-core/src/main/java/water/api/FramesHandler.java

```diff
@@ -129,7 +105,7 @@
   @SuppressWarnings("unused") // called through reflection by RequestServer
   public FramesV3 list(int version, FramesV3 s) {
     Frames f = s.createAndFillImpl();
-    f.frames = Frames.fetchAll();
+    f.frames = Frame.fetchAll();
```

Comment:
Yes, because I'm now using it in StackedEnsemble, not just from a REST API handler.